x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -441,6 +443,9 @@
     // Recompile if you want to compare performance with C++ tokenization.
     public static final boolean CATEGORIZATION_TOKENIZATION_IN_JAVA = true;
 
+    public static final LicensedFeature.Persistent ML_JOBS_FEATURE =
+        LicensedFeature.persistent("machine-learning-job", License.OperationMode.PLATINUM);
```

Comment:
```suggestion
        LicensedFeature.persistent("machine-learning-anomaly-detection-job", License.OperationMode.PLATINUM);
```
Since we also have `data-frame-analytics-job` :D